pkg/operator/operator.go

```diff
@@ -857,6 +857,9 @@ func (o *Operator) loadConfig(key string) (*manifests.Config, error) {
 	}
 
 	cParsed, err := manifests.NewConfigFromString(configContent)
+	if tp {
+		cParsed, err = manifests.NewConfigFromStringTp(configContent)
+	}
```

Comment:
Nit: This code looks a little awkward to me. The first `err` might not actually be looked at. This not an issue now, since it runs the same code right now (lets ignore the inefficiency), but I can imagine future changes might make this less obvious. It might be better here to either separate the two calls with `if...else...` or change the signature of the new function to `func NewConfigFromStringTp(content string, tp bool) (*Config, error)` and expose the tech preview flag.

With the second option we could also add a comment to `NewConfigFromString` that this is deprecated and one should use `NewConfigFromStringTp(content, false)` as a drop in replacement. This gets rid of this little awkwardness.

Reply:
Yeah, I agree, I was not too fond of it either. I think I'll follow the second approach as I believe that `NewConfigFromString` should always receive the `tp` argument since in the future there will be other features that are tech preview and will face a similar issue